xapian-bindings/python3/generate-letor-python-exceptions

```diff
@@ -0,0 +1,165 @@
+# generate-letor-python-exceptions: generate error handling code for Python bindings
+my $copyright = <<'END';
+ Copyright (C) 2003,2004,2006,2007,2008,2009,2012 Olly Betts
+ Copyright (C) 2007 Lemur Consulting Ltd
+ Copyright (C) 2007 Richard Boulton
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License as
+ published by the Free Software Foundation; either version 2 of the
+ License, or (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+END
+
+use strict;
+use exception_data;
+
+my @allclasses = (@baseclasses, @classes);
+
+my @posting_source_virtual_methods = qw(
+    get_termfreq_min
+    get_termfreq_est
+    get_termfreq_max
+    get_maxweight
+    get_weight
+    next
+    skip_to
+    check
+    at_end
+    get_docid
+    get_description
+    init
+    name
+);
+
+my @matchspy_virtual_methods = qw(
+    name
+    merge_results
+    get_description
+);
```

Comment:
The above two lists are specific to the xapian-core API.

Reply:
fixed in a fixup commit.